test/Microsoft.DotNet.Cli.Utils.Tests/BuiltInCommandTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class BuiltInCommandTests : TestBase
+    {
+        [Fact]
+        public void TestExecute()
```

Comment:
Can you put the expectation of the result of this test in the method name? 

Having to go through and change a bunch of tests this week, I've found being able to determine what the test is supposed to do from the method name is immensely helpful.